Framework/Core/src/ExternalFairMQDeviceProxy.cxx

```diff
@@ -598,6 +624,9 @@ DataProcessorSpec specifyExternalFairMQDeviceProxy(char const* name,
               timingInfo.creation = dph->creation;
             }
             dataHandler(timingInfo, parts, 0, ci);
+            if (ctx.services().active<o2::monitoring::Monitoring>()) {
```

Comment:
Can you please use a boolean flag which defaults to false so that we do not change the behaviour for all the proxies we have? I fear with Monitoring the overhead of sending this message online is too high.

Reply:
Done.